torch_geometric/nn/conv/fused_gat_conv.py

```diff
@@ -0,0 +1,132 @@
+from typing import Tuple, Union
```

Comment:
I think we should be able to inherit from `GATConv` here, and just override its `forward` method. Correct?

Reply:
Yeah, I think so. We can inherit from GATConv instead.